components/core/tools/scripts/lib_install/musllinux_1_2/install-prebuilt-packages.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+
+apk update && apk add --no-cache \
+    openjdk11-jdk \
+    jq \
+    curl-dev \
+    bzip2-dev \
+    bzip2-static \
+    mariadb-connector-c-dev \
+    openssl-dev \
+    zlib-dev \
+    zlib-static
```

Comment:
```suggestion
    bzip2-dev \
    bzip2-static \
    curl-dev \
    jq \
    mariadb-connector-c-dev \
    openjdk11-jdk \
    openssl-dev \
    zlib-dev \
    zlib-static
```
To follow the alphabetical ordering